### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by robertfoss** [ferristseng/rust-lifx#1]
Hey,

I just checked out this library and it seems to be having issues compiling on the current rust stable.
```
error: no rules expected the token `flags`
  --> /home/robertfoss/.cargo/git/checkouts/rust-lifx-b19e8b8eea76146f/a8e562d/src/client.rs:63:7
   |
63 |   pub flags DiscoverOptions: u8 {
   |       ^^^^^

error: Could not compile `lifx`.
```

```
$ rustc --version
rustc 1.21.0 (3b72af97e 2017-10-09)
```

Apart from it not building this looks like the by nicest lifx rust lib.

**Comment by jibaku** [comment]
Should me resolved in master now

**Comment by ferristseng** [comment]
Fixed from #2.
